### ISSUE TITLE
[crash] PRINT with block argument crashes in GUI console

### ISSUE THREAD

**Issue by Oldes** [red/red#3857]
**Describe the bug**
Including red-system code in Red, which contains `print-line` with a `block!` argument crashes GUI console,but is working fine under CLI console. 

**To reproduce**
Include this in `gui-console.red` file:
```
test: context [
	foo: routine[][
		print-line "hello" ;<--- this one is always fine
		print-line ["--> " 1] ;<--- this crashes in GUI console but not with CLI
	]
]
```
and call `test/foo` from console.

**Expected behavior**
Line with text is printed. Now it crash under GUI.

**Comment by qtxie** [comment]
I cannot reproduce it. No crash on Win10.
```
>> about
Red 0.6.4 for Windows built 21-Apr-2019/15:04:09+02:00

>> test/foo
hello
--> 1
```

**Comment by Oldes** [comment]
@qtxie are you using release mode? The one when there is no CLI window open.

**Comment by qtxie** [comment]
@qtxie OK. With `-t Windows` flag I can reproduce the issue now.

**Comment by qtxie** [comment]
Fixed in commit 9c7cd2b
